dotnet-core-webapi/HoneycombMiddleware.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace dotnet_core_webapi_sample
+{
+  public class HoneycombMiddleware
+  {
+    private readonly RequestDelegate _next;
+    private readonly HttpClient _client;
+    public HoneycombMiddleware(RequestDelegate next)
+    {
+      _next = next;
+      _client = new HttpClient();
+      _client.DefaultRequestHeaders.Add("X-Honeycomb-Team","6b7919e9cb69152d49b8bd3aa1fffbc4");
```

Comment:
do we include a write key like this in our other examples?

Reply:
facepalm.